.github/workflows/tox.yml

```diff
@@ -179,6 +179,11 @@ jobs:
       with:
         name: logs.zip
         path: .tox/**/log/
+    - name: Report junit failures
+      uses: shyim/junit-report-annotations-action@master
```

Comment:
I'm not sure about using `master`. Should we wait till https://github.com/shyim/junit-report-annotations-action/issues/3#issuecomment-663261847 gets addressed and pin it then?

Reply:
Is ok, the risk is very low. The token is read-only.

Reply:
My concern is not (only) about that:
1) The action may suddenly break the interface.
2) The token only has read-only mode for PR runs, there's still cron runs and tag creations that will run in-repo and have higher privileges.